typings/index.d.ts

```diff
@@ -39,6 +39,17 @@ declare module "noblox.js" {
         IsPrimary: boolean,
     }
 
+    interface GroupGameInfo {
+        id: number;
+        name: string;
+        description: string | null;
```

Comment:
`description?` maybe?

Reply:
Fixed and I corrected the types for some other interfaces that used null unions instead of optional parameters.